.gitignore

```diff
@@ -102,3 +102,8 @@ ENV/
 
 # pytest
 .pytest_cache/
+
```

Comment:
Please add editor specific ignores to you global git ignore. Not to a repository.